### ISSUE TITLE
New feature request for Sprite handling called "created"

### ISSUE THREAD

**Issue by toetoast** [Bodmer/TFT_eSPI#682]
Hi, this is not an issue but more of an enhancement for consideration.

I've been using the following new Method (called "created") for the **TFT_eSprite** Class, that I think others may find useful:

```
/***************************************************************************************
** Function name:        created
** Description:             Return the creation status of the Sprite
****************************************************************************************/
bool TFT_eSprite::created(void)
{
   if (_created) return true;
   else return false; 
}
```
Here's an example of how I use it within a function.  Here "ImageToSprite" loads a BMP file into a Sprite that was defined earlier. The pointer to the Sprite is passed:

`ImageToSprite("woof.bmp", &spr01)`

Within the function the existence of the Sprite is checked before proceeding with the rest of the function:

```
bool ImageToSprite(char *FileName, TFT_eSprite *_spr)
{
    if (!_spr->created())
    {
        Serial.print("Undefined Sprite");
        return false;
    }
}
```
In summary, this provides a way to place some error checking in your code.

Cheers

**Comment by Bodmer** [comment]
Done, thanks.
